src/wizards/newspack/views/settings/theme-and-brand/utils.ts

```diff
@@ -99,36 +99,51 @@ const MONOSPACE_FONTS = [
 	{ label: 'Space Mono' },
 	{ label: 'Roboto Mono' },
 ];
-const ALL_FONTS = [ ...SERIF_FONTS, ...SANS_SERIF_FONTS, ...DISPLAY_FONTS, ...MONOSPACE_FONTS ];
+const ALL_FONTS = [
+	...SERIF_FONTS,
+	...SANS_SERIF_FONTS,
+	...DISPLAY_FONTS,
+	...MONOSPACE_FONTS,
+];
+
+export const TYPOGRAPHY_OPTIONS: { value: TypographyOptions; label: string }[] =
+	[
+		{ value: 'curated', label: __( 'Default', 'newspack-plugin' ) },
+		{ value: 'custom', label: __( 'Custom', 'newspack-plugin' ) },
+	];
 
-export const getFontsList = ( headingsOnly: boolean ) =>
+export const getFontsList = ( headingsOnly: boolean = false ) =>
 	[
 		{
-			label: __( 'Serif', 'newspack' ),
+			label: __( 'Serif', 'newspack-plugin' ),
 			fallback: 'serif',
 			options: SERIF_FONTS,
 		},
 		{
-			label: __( 'Sans Serif', 'newspack' ),
+			label: __( 'Sans Serif', 'newspack-plugin' ),
 			fallback: 'sans_serif',
 			options: SANS_SERIF_FONTS,
 		},
 		{
-			label: __( 'Display', 'newspack' ),
+			label: __( 'Display', 'newspack-plugin' ),
 			fallback: 'display',
 			options: DISPLAY_FONTS,
 		},
 		{
-			label: __( 'Monospace', 'newspack' ),
+			label: __( 'Monospace', 'newspack-plugin' ),
 			fallback: 'monospace',
 			options: MONOSPACE_FONTS,
 		},
 	]
-		.map( group => ( { ...group, options: processFontOptions( headingsOnly, group.options ) } ) )
+		.map( group => ( {
+			...group,
+			options: processFontOptions( headingsOnly, group.options ),
+		} ) )
 		.filter( group => group.options.length );
 
 export const isFontInOptions = ( label: string ) =>
-	ALL_FONTS.filter( option => option.label.indexOf( label ) === 0 ).length >= 1;
+	ALL_FONTS.filter( option => option.label.indexOf( label ) === 0 ).length >=
+	1;
```

Comment:
Was this a linting change? 😄

Reply:
I have lint configured to run on `save` and missed this quirk. I'd have to pause the auto-saving to get around this so I replaced:
`ALL_FONTS.filter( option => option.label.indexOf( label ) === 0 ).length >= 1;` 
with
`ALL_FONTS.filter( option => ! option.label.includes( label ) ).length >= 1;`
instead 🤓